### ISSUE TITLE
Should aria-tooltip-name fail if it's not focusable?

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#4395]
I'm not sure the following is actually a failure of WCAG.

```html
<button onhover="toggle('#tooltip')">
  button
</button>
<div role="tooltip">
  <span id="tooltip" hidden> Hello world </span>
</div>
```

VoiceOver + Safari seems to ignore the empty tooltip, which I think it what it should do while not hovered. If other screen readers also ignore empty tooltips I don't think axe should fail them. Its worth testing if there's a difference when the tooltip is focusable. I assume that will announce the empty tooltip, which would be a problem.

Tooltips shouldn't normally be focusable so I'm a little conflicted about whether we should make this rule a best pracitce, or if this should incomplete when not focusable. Open for discussion, but we should run some AT tests first.
